addons/mail/static/src/models/file_uploader_view/file_uploader_view.js

```diff
@@ -4,10 +4,147 @@ import { registerModel } from '@mail/model/model_core';
 import { attr, many2one, one2one } from '@mail/model/model_field';
 import { clear, replace } from '@mail/model/model_field_command';
 
+import core from 'web.core';
+
+const geAttachmentNextTemporaryId = (function() {
+    let tmpId = 0;
+    return () => {
+        tmpId += 1;
+        return tmpId;
+    };
+})();
```

Comment:
What? Am I missing something? That looks terribly wrong (even in existing code), the ID should be negatively incremented, otherwise it will overlap with existing real ids.

We probably are very lucky that a lot of low id attachments are stuff created by the framework that the user never tries to access.

Try to replicate a crash, and if it's possible, this should be fixed in v14.